src/config.ts

```diff
@@ -1,10 +1,29 @@
-import { ColorResolvable, ComponentEmojiResolvable, PermissionsBitField } from "discord.js";
+import { ColorResolvable, ComponentEmojiResolvable } from "discord.js";
 
 // Configure your bot here.
 export const config: BotConfig = {
+	// --- General ---
 	botName: "Jambo",
 	iconURL: "https://raw.githubusercontent.com/Cowoding-Jams/Jambo/main/images/Robot-lowres.png",
 	githubURL: "https://github.com/Cowoding-Jams/Jambo",
+	serverDescription:
+		"We're a group of young and mostly queer people having a game jam/hackathon server together. We're a very friendly and welcoming community and are happy to have you join us! \nCheck out <#1022874504525008997> for more information!",
+	color: "#F0A5AC",
+
+	// --- Development ---
+	logLevel: "debug",
+
+	// --- Management ---
+	adminRoleId: "855348792025939988", // cowoding-jams/roles: admin
+	moderatorRoleId: "855349314807922698", // cowoding-jams/roles: moderator
+
+	// --- Polls & Coding Jams ---
+	jamRoleName: "jammin",
+	pollChannelId: "1063379924267847761", // cowoding-jams/channels: coding-jams
+	jamChannelId: "1063379924267847761", // cowoding-jams/channels: coding-jams
+	resultCategoryId: "855356830724259871", // cowoding-jams/categories: Working Area
```

Comment:
Thinking about this, doesn't it make more sense to leave this type of config empty? Can Imagen that someone has trouble because a channel is wrong but all fields are filled in

Reply:
I don't understand